seed/tests/test_column_mapping_profiles_views.py

```diff
@@ -120,7 +118,7 @@ def test_update_preset_endpoint(self):
         self.assertEqual('changed_preset_name', datum['name'])
         self.assertEqual(1, ColumnMappingPreset.objects.filter(name='changed_preset_name').count())
 
-    def test_create_preset_endpoint(self):
+    def test_create_profile_endpoint(self):
         url = reverse('api:v2:column_mapping_presets-list') + '?organization_id=' + str(self.org.id)
```

Comment:
I do think we want to convert this one to use v3. Not sure how this will react when it can't find the `list` endpoint... Maybe  `api:v2:column_mapping_presets-list` might have to change to `api:v2:column_mapping_presets-create`?